ui/apps/platform/src/Containers/Integrations/utils/integrationUtils.ts

```diff
@@ -13,6 +14,7 @@ export type Integration = {
     type: IntegrationType;
     id: string;
     name: string;
+    traits?: Traits;
```

Comment:
Too bad, so sad that there are multiple sources of truth. Thank you for careful work.